src/osa/tests/test_gainsel_webmaker.py

```diff
@@ -0,0 +1,16 @@
+def test_gain_selection_webmaker(
```

Comment:
you can add this test in the existing test script file https://github.com/cta-observatory/lstosa/blob/main/src/osa/scripts/tests/test_osa_scripts.py No need for a new file